client/src/components/LoginAndReg/Login.js

```diff
@@ -13,16 +13,20 @@ const Login = (props) => {
   const [email, setEmail] = useState('')
   const [password, setPassword] = useState('')
   const [error, setError] = useState('')
-  const [success, setSuccess] = React.useState(false);
-  const [msg, setMsg] = React.useState('');
+  const [success, setSuccess] = useState(false);
+  const [msg, setMsg] = useState('');
   const { request_error } = useSelector(state => state.network);
 
   useEffect(() => {
     if (request_error && request_error.error) {
       setMsg(request_error.error)
       setError(true);
     }
-  }, [])
+    else {
+      setMsg('');
+      setError(false);
+    }
+  }, [request_error])
```

Comment:
This part is needed. Thank you for adding this. I forgot to add the request_error to the dependency list.